Python/libraries/recognizers-date-time/recognizers_date_time/date_time/base_holiday.py

```diff
@@ -233,17 +240,17 @@ def __init__(self):
 
     def _init_holiday_funcs(self) -> Dict[str, Callable[[int], datetime]]:
         return dict([
-            ('fathers', BaseHolidayParserConfiguration.fathers_day),
-            ('mothers', BaseHolidayParserConfiguration.mothers_day),
-            ('thanksgivingday', BaseHolidayParserConfiguration.thanksgiving_day),
-            ('thanksgiving', BaseHolidayParserConfiguration.thanksgiving_day),
-            ('blackfriday', BaseHolidayParserConfiguration.black_friday),
-            ('martinlutherking', BaseHolidayParserConfiguration.martin_luther_king_day),
-            ('washingtonsbirthday', BaseHolidayParserConfiguration.washingtons_birthday),
-            ('labour', BaseHolidayParserConfiguration.labour_day),
-            ('canberra', BaseHolidayParserConfiguration.canberra_day),
-            ('columbus', BaseHolidayParserConfiguration.columbus_day),
-            ('memorial', BaseHolidayParserConfiguration.memorial_day)
+            (Constants.FATHERS, BaseHolidayParserConfiguration.fathers_day),
```

Comment:
Ideally we should have a way to define this type of initialization from the YAML files too. But this can be its own separate issue.

Reply:
Same as before.

Reply:
Please create a new issue for this. It would be great if the new init solution can be applied to all platforms at once.

Reply:
Sure thing, we'll create that issue today and link it to this PR